src/appmetrics.cpp

```diff
@@ -528,8 +544,11 @@ NAN_METHOD(sendControlCommand) {
     }
 
     if (info[0]->IsString() && info[1]->IsString()) {
-        String::Utf8Value topicArg(info[0]->ToString());
-        String::Utf8Value commandArg(info[1]->ToString());
+        Isolate* isolate = v8::Isolate::GetCurrent();
```

Comment:
`isolate` not used in this function.

Reply:
thanks, this and a few other things is probably left over from my fixes before I switched some over to use their `Nan` equivalents. Will remove in a commit